src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -159,12 +159,16 @@ private function configBuilder(string $namespace): ConfigBuilderInterface
 
         // If it does not start with Symfony\Config\ we dont know how to handle this
         if ('Symfony\\Config\\' !== substr($namespace, 0, 15)) {
-            throw new InvalidargumentException(sprintf('Could not find or generate class "%s".', $namespace));
+            throw new InvalidArgumentException(sprintf('Could not find or generate class "%s".', $namespace));
         }
 
         // Try to get the extension alias
         $alias = Container::underscore(substr($namespace, 15, -6));
 
+        if (false !== strpos($alias, '\\')) {
+            throw new InvalidArgumentException(sprintf('Nested bundle config class "%s" could not be handled.', $namespace));
+        }
+
         if (!$this->container->hasExtension($alias)) {
```

Comment:
Is it possible for an extension to be aliased: `foobar\config`?

It is not the "standard way", but I dont know why it would not be allowed... Maybe first check if the container has the extension, and then throw this new exception if the alias includes a backslash?

Reply:
I think an alias may contain any symbols, but it ConfigBuilder class will not contain `\` anyway after normalization.

Reply:
That is true. If the extension alias is `foobar\acme`, we would find that extension, generate the ConfigBuilder as `Symfony\Config\FoobarConfig` and the user would get an error like: 

```
Callback expects Symfony\Config\Foobar\AcmeConfig but Symfony\Config\FoobarConfig given. 
```

You are correct. We should not move this.